pages/in/widget/pay/index.vue

```diff
@@ -439,40 +440,50 @@ export default {
           paymentRedirectWhiteList,
         };
       });
+      let redirectWhitelistError = null;
+      let isRedirectURLWhiteListed = false;
       if (agentUser && !agentUser.cosmosWallet) {
-        error({ statusCode: 400, message: 'VIA_USER_HAS_NO_WALLET' });
+        redirectWhitelistError = redirectWhitelistError || { statusCode: 400, message: 'VIA_USER_HAS_NO_WALLET' };
       }
-
       if (toUsers.some(u => !u.cosmosWallet)) {
-        error({ statusCode: 400, message: 'RECEIPIENT_HAS_NO_WALLET' });
+        redirectWhitelistError = redirectWhitelistError || { statusCode: 400, message: 'RECEIPIENT_HAS_NO_WALLET' };
       }
```

Comment:
Please do not modify these error unrelated to `redirectUri`, just throw them directly